### DISCUSSION TITLE
Changing part of the URL

### DISCUSSION THREAD

**Discussion by gchickengod** [requestly/requestly#299] [General]
How to create a universal redirection that would change only the IP in the address bar, and the rest would remain the same? For example https://blabla.com/*  replace with http://192.168.0.1/*  
So that this rule would work with any text under *

**Comment by lazyvab** [comment]
@gchickengod This is a very common use-case and easily solved using `Replace String` rule.
https://app.requestly.io/rules/editor/create/Replace

You need to define a condition where `https://blabla.com` will be replaced with `http://192.168.0.1` in any URL. The other part of the URL will remain the same.

<img width="902" alt="Screenshot 2023-02-01 at 7 38 03 PM" src="https://user-images.githubusercontent.com/6367566/216065315-bea9ba63-0d09-409d-98e7-023973ac6ba3.png">

**Comment by gchickengod** [comment]
> @gchickengod This is a very common use-case and easily solved using `Replace String` rule. https://app.requestly.io/rules/editor/create/Replace
> 
> You need to define a condition where `https://blabla.com` will be replaced with `http://192.168.0.1` in any URL. The other part of the URL will remain the same.
> 
> <img alt="Screenshot 2023-02-01 at 7 38 03 PM" width="902" src="https://user-images.githubusercontent.com/6367566/216065315-bea9ba63-0d09-409d-98e7-023973ac6ba3.png">

Thanks, it works. I wouldn't have found the right window without your link. :)

**Comment by sachinjain024** [comment]
@lazyvab We should cover this use-case as a FAQ in our upcoming documentation revamp.

**Comment by lazyvab** [comment]
@gchickengod If you click on "New Rule" button on https://app.requestly.io/rules/my-rules page, you will find all rule type options. You need to select `Replace String` and click `Create Rule`.

**Comment by ashuvax** [comment]
Is it possible to route all traffic from one address to a second address for example:
from https://*
To https://sjj.com/{here insert the original address}

**Comment by wrongsahil** [comment]
@ashuvax Does`https://*` here means that  want to match all domains?  or you want to route traffic for a single domain and just match rest of the path? 

If you want for only certain domains like this `https://abc.com/*` -> `https://sjj.com/*`. Then this should be possible using `Replace Rule`. Just replace `abc.com` with `sjj.com`

**Reply by ashuvax** [reply]
i'm want to match all domains

**Reply by wrongsahil** [reply]
You can use `Redirect Rule` with wildcard matching for this. 

![image](https://user-images.githubusercontent.com/16779465/218056858-fb02a441-1874-4654-a5f8-5932c72c4a91.png)

**Reply by ashuvax** [reply]
The problem is that it goes into a loop all the time without a break
Is it possible to write to him to skip sjj.com?
